tests/manage/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -106,8 +126,19 @@ def setup_teardown(self, request):
         self.pv_obj = ocp.OCP(kind=constants.PV, namespace=self.namespace)
         self.nfs_sc = "ocs-storagecluster-ceph-nfs"
         self.sc = ocs.OCS(kind=constants.STORAGECLASS, metadata={"name": self.nfs_sc})
-        self.test_folder = "test_nfs"
-        utils.exec_cmd(cmd="mkdir -p " + self.test_folder)
+        self.run_id = config.RUN.get("run_id")
+        self.test_folder = f"mnt/test_nfs_{self.run_id}"
+        log.info(f"nfs mount point out of cluster is----- {self.test_folder}")
+        self.nfs_client_ip = config.ENV_DATA["nfs_client_ip"]
+        log.info(f"nfs_client_ip is: {self.nfs_client_ip}")
+
+        self.nfs_client_user = config.ENV_DATA["nfs_client_user"]
+        log.info(f"nfs_client_user is: {self.nfs_client_user}")
+
+        self.nfs_client_pwd = config.ENV_DATA["nfs_client_pwd"]
+        log.info(f"nfs_client_pwd is: {self.nfs_client_pwd}")
+
+        self.con = ""
```

Comment:
```suggestion
        self.con = None
```
This looks better to me to use None instead of empty string


Or why not to init the connection directly here on one place instead of in each of test case?